frontendUiApi/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/table/BaseGraphTable.java

```diff
@@ -19,7 +19,8 @@
   protected BaseGraphTable(TableModel model) {
     super(model);
 
-    // Without set autoresize off column will expand to whole table width (will not fit the content size)
+    // Without set autoresize off column will expand to whole table width (will not fit the content size).
+    // This cause the branch tooltips (sync to parent status descriptions) are displayed on the whole Git Machete panel width
```

Comment:
```suggestion
    // This causes the branch tooltips (sync to parent status descriptions) to be displayed on the whole Git Machete panel width instead of just over the text.
```
(I think it should be `over the text` ??)